templates/shared/_download-newsletter.html

```diff
@@ -3,33 +3,131 @@ <h2>Sign up for our newsletter</h2>
   <p>Get the latest Ubuntu news and updates in your inbox.</p>
 </div>
 
-<form action="/marketo/submit" method="post" id="mktoForm_4960" onsubmit="ga('send', 'Newsletter', 'Signup', '{{ ga_event_title }} newsletter signup');">
+<form action="/marketo/submit"
+      method="post"
+      id="mktoForm_5883"
+      onsubmit="stringifyCustomFields(); ga('send', 'Newsletter', 'Signup', '{{ ga_event_title }} newsletter signup');">
   <label for="email">Email*:</label>
-  <input required id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" />
-
+  <input required
+         id="email"
+         name="email"
+         maxlength="255"
+         type="email"
+         pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" />
+  {% if request.path == '/download/desktop/thank-you' %}
+    <textarea class="u-hide"
+              id="ubuntu_desktop_usage"
+              name="UbuntuDesktopPlannedUsage"
+              rows="5"
+              maxlength="2000"></textarea>
+    <label for="ubuntu_desktop_usage_fields" id="ubuntu_desktop_usage">How do you plan to use Ubuntu Desktop?</label>
+    <div class="row p-section--shallow" id="ubuntu_desktop_usage_fields">
+      <div class="col-2">
+        <label class="p-checkbox js-checkbox">
+          <input type="checkbox" aria-labelledby="work" class="p-checkbox__input" />
+          <span class="p-checkbox__label" id="work">Work</span>
+        </label>
+      </div>
+      <div class="col-2">
+        <label class="p-checkbox js-checkbox">
+          <input type="checkbox" aria-labelledby="education" class="p-checkbox__input" />
+          <span class="p-checkbox__label" id="education">Education</span>
+        </label>
+      </div>
+      <div class="col-2">
+        <label class="p-checkbox js-checkbox">
+          <input type="checkbox" aria-labelledby="personal" class="p-checkbox__input" />
+          <span class="p-checkbox__label" id="personal">Personal use</span>
+        </label>
+      </div>
+    </div>
+    <hr />
+  {% endif %}
   <div class="p-section--shallow">
     <label class="p-checkbox">
-      <input required class="p-checkbox__input" value="yes" aria-labelledby="canonicalUpdatesOptIn" name="canonicalUpdatesOptIn" type="checkbox">
+      <input required
```

Comment:
```suggestion
      <input required
```
Does this need to be required? Receiving marketing information is usually optional

Reply:
I guess it should be as the user is signing up for a newsletter therefore should agree to receive it. @Sophie-32 what do you think?

Reply:
Yes, required since its a newsletter sign up (but pete's correct that _usually_ it is optional)